src/Forms/SubmitHours.tsx

```diff
@@ -17,26 +17,35 @@ type FormValues = {
 export default function Command({
   projectId,
   taskId,
+  timeEntryId,
   hours = "7.5",
   initialDate = new Date(),
+  notes = null,
   skipToSubmit = false,
 }: {
   projectId: number;
   taskId: number;
+  timeEntryId?: number;
   hours?: string;
   initialDate?: Date;
+  notes?: string | null;
   skipToSubmit?: boolean;
 }) {
   const { push } = useNavigation();
   const { handleSubmit, itemProps, setValidationError, values } = useForm<FormValues>({
     initialValues: {
       spent_date: initialDate,
       hours: hours,
+      notes: notes ?? undefined
```

Comment:
Maybe the `notes` in `FormValues` can be updated to be of type `string | null`, then we can just pass `notes` as is.

Reply:
Good point :+1: Only issue is that the `value` prop in `Form.TextArea` only accepts `string | undefined` 🙃

Reply:
Ah, nevermind me then 👍

Reply:
So you might have to do something like this instead

<img width="1209" alt="image" src="https://github.com/user-attachments/assets/4b95f011-839e-41a3-a32e-1cdfa91bbc86">

Reply:
🤮